cheeseshop/dbapi.py

```diff
@@ -323,8 +392,103 @@ class CsGoDeathEvent(object):
             CREATE TABLE cs_go_death_events(
                 id serial PRIMARY KEY,
                 attacker integer REFERENCES cs_go_steam_ids (id),
+                attacker_pos_x real,
+                attacker_pos_y real,
+                attacker_pos_z real,
                 victim integer REFERENCES cs_go_steam_ids (id),
-                weapon text
+                victim_pos_x real,
+                victim_pos_y real,
+                victim_pos_z real,
+                assister integer REFERENCES cs_go_steam_ids (id),
+                weapon_original_owner integer REFERENCES cs_go_steam_ids (id),
+                penetrated smallint,
+                weapon text,
+                map_name text,
```

Comment:
Why do we need this if we have the cs_go_map releation?

Reply:
We don't. Will remove.